source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/frame_transformer/frame_transformer_cfg.py

```diff
@@ -31,9 +31,13 @@ class FrameCfg:
         """Information specific to a coordinate frame."""
 
         prim_path: str = MISSING
-        """The prim path corresponding to the parent rigid body.
+        """The prim path corresponding to a rigid body.
 
-        This prim should be part of the same articulation as :attr:`FrameTransformerCfg.prim_path`.
+        This can be a regex pattern to match multiple prims. For example, "/Robot/.*" will match all prims under "/Robot".
+
+        This means that if the source `prim_path` is "/Robot/base", and the target `prim_path` is "/Robot/.*",
+        then the FrameTransformer will track the pose of all prims under "/Robot" in the source frame,
+        including "/Robot/base" (even though this will result in an identity pose w.r.t. the source frame).
```

Comment:
Was a bit confusing what is meant by source/target here. Added attribute names to make it clearer

```suggestion
        This means that if the source :attr:`FrameTransformerCfg.prim_path` is "/Robot/base", and the target :attr:`FrameTransformerCfg.FrameCfg.prim_path` is "/Robot/.*",
        then the frame transformer will track the poses of all the prims under "/Robot",
        including "/Robot/base" (even though this will result in an identity pose w.r.t. the source frame).
```